src/plugins/lights/public/js/lights.js

```diff
@@ -32,47 +32,48 @@
             }
 
             // Setup input handlers
-            this.inputDefaults = 
-            [
-              {
+            this.inputDefaults = [
+            {
                 name: 'plugin.lights.stepPositive',
                 description: 'Makes the ROV lights brighter.',
-                defaults: 
-                {
-                  keyboard: 'p',
-                  gamepad: 'DPAD_UP'
-                },
-                down: function () 
+                shortName: "Increase ROV Light Brightness",
```

Comment:
After looking at it again, I agree with you. This was in a response to wanting a **Command** section that describes what the command is doing, in brief, so you could quickly see and understand what you are changing in the input configuration page. Will remove.